src/scripts/main.js

```diff
@@ -1,17 +1,39 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const block = document.querySelector('body');
+
+  const message = document.createElement('div');
+  const h2 = document.createElement('h2');
+  const p = document.createElement('p');
+
+  message.style.top = `${posTop}px`;
+
+  message.style.right = `${posRight}px`;
+
+  message.classList.add('notification', type);
+
+  h2.classList.add('title');
+
+  h2.innerText = title;
+
+  p.innerText = description;
+
+  block.children[0].after(message);
+
+  message.append(h2, p);
+
+  setTimeout(() => message.remove(message), 2000);
```

Comment:
```suggestion
  setTimeout(() => message.remove(), 2000);
```
actually, it doesn't accept any arguments. the method just deletes the element on which it was invoked